en/docs/develop/tools/integration-tools/persist-tool.md

```
@@ -165,13 +165,113 @@ stream<Employee, error?> employees = sClient->/employees;
+| `migration-label` | Yes | — | A descriptive label for the migration (for example, `add_email_column`) |
+| `--datastore` | Yes | — | Target data store: `mysql`, `mssql`, or `postgresql` |
+| `--module` | Yes | — | Name of the module containing the data model |
+
+```bash
+# Create a migration after modifying the data model
+bal persist migrate --datastore mysql --module db add_phone_column
+```
+
+The command generates a timestamped migration directory under `persist/migrations/`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)